<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerDataWidget</class>
 <widget class="QWidget" name="CustomerDataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="3" column="0">
      <widget class="QLabel" name="lbMobilePhone">
       <property name="text">
        <string>Mobile Phone +33674801256</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="lbNbBill">
       <property name="text">
        <string>25 Factures</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="lbNbDev">
       <property name="text">
        <string>12 Devis</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lbAddress">
       <property name="text">
        <string>1 rue du trou du cul </string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="lbPostalCodeCityCountry">
       <property name="text">
        <string>12345 MACHINCHOSE</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lbEmail">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;#&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;a.A@truc.com&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbName">
       <property name="text">
        <string>Monsieur a A</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="lbNbProject">
       <property name="text">
        <string>42 projets</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lbCompany">
       <property name="text">
        <string>&lt;b&gt;Organisation A&lt;/b&gt;</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lbFax">
       <property name="text">
        <string>Fax +33674801256</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbPhone">
       <property name="text">
        <string>Phone +33674801256</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
